must_gather.sh

```diff
@@ -18,6 +18,7 @@ if [ -n "${MIRROR_IMAGES}" ]; then
 
   OPENSHIFT_RELEASE_VERSION=$(oc adm release info --registry-config="$pullsecret_file" "$OPENSHIFT_RELEASE_IMAGE" -o json | jq -r ".config.config.Labels.\"io.openshift.release\"")
   MUST_GATHER_IMAGE="--image=${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image:${OPENSHIFT_RELEASE_VERSION}-must-gather"
+  rm -f "$pullsecret_file"
```

Comment:
It would be better to remove this with a trap ref https://github.com/openshift-metal3/dev-scripts/pull/936